lib/rubocop/cop/rails/index_true.rb

```diff
@@ -0,0 +1,39 @@
+module RuboCop
+  module Cop
+    module Rails
+      # check that index: true is not being used in migrations
+      #
+      # @example
+      #   # bad
+      #   add_column :booking_templates, :booking_id, :integer
+      #
+      #   # good
+      #   add_index :booking_templates, :booking_id
+      #
+      class IndexTrue < Cop
+        MSG = '
+          index: true does not work in an add_column or change_column statement
```

Comment:
All code examples should be in backticks here to get colored in the output. I'd use some heredoc for this long string.